src/qtt/algorithms/functions.py

```diff
@@ -435,17 +436,23 @@ def estimate_dominant_frequency(signal, sample_rate=1, remove_dc=True, fig=None)
 
     if remove_dc:
         w[0] = 0
+    w[freqs < 0] = 0
```

Comment:
`w` is the fft of the input signal. I don't want to use `fft` (since that is already the method performing the transform). Would `signal_fft` be better?